tgs/src/main/cpp/tgs/FeatureReduction/PrincipalComponentsAnalysis.cpp

```diff
@@ -31,13 +31,13 @@
 #include <assert.h>
 #include <iostream>
 
-#include "../RandomForest/DataFrame.h"
-#include "../StreamUtils.h"
+#include <tgs/StreamUtils.h>
+#include <tgs/RandomForest/DataFrame.h>
```

Comment:
In the TGS library I got rid of a bunch of relative paths because I didn't like them and I was on a tear.